course/src/main/java/org/openedx/course/presentation/container/CourseContainerViewModel.kt

```diff
@@ -109,23 +109,33 @@ class CourseContainerViewModel(
         }
     }
 
-    fun courseTabClickedEvent() {
+    fun courseTabClickedEvent(tab: CourseContainerTab) {
```

Comment:
Can we rename it slightly? It might impact readability since we already have a tab named 'Course'.